### DISCUSSION TITLE
Marketing

### DISCUSSION THREAD

**Discussion by CodeDoctorDE** [LinwoodDev/Butterfly#61] [General]
Do you have any ideas how I can promote this app?

**Comment by Shahin-rmz** [comment]
You want to monetize or just promote?

**Reply by CodeDoctorDE** [reply]
I want to promote the app. Monetization is currently no point on my to-do list

**Reply by Shahin-rmz** [reply]
at a time I lived in Germany, I was participant of a group of Linux users, where we spread the words about FOSS Programms.
Now I will talk to some influencer/YouTube video makers to promote the app.
You can also make a group in Telegram or discord, where non tech savy users of the app can gather and negotiate.

**Reply by CodeDoctorDE** [reply]
Thanks my discord is here: https://discord.gg/97zFtYN
